lib/java/com/google/android/material/lists/viewHolders/singleLine/SingleLineCircleItemViewHolder.java

```diff
@@ -0,0 +1,46 @@
+package com.google.android.material.lists.viewHolders.singleLine;
+/*
+ * Copyright (C) 2019 The Android Open Source Project
```

Comment:
Should the year be updated to 2020? If so, the suggestion also applies to the other Java source files below.